domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -439,4 +498,137 @@ public static void deleteById(String table, String id,
   public static PostgresClient postgresClient(Context vertxContext, Map<String, String> okapiHeaders) {
     return PostgresClient.getInstance(vertxContext.owner(), TenantTool.tenantId(okapiHeaders));
   }
+  /**
+   * Generate a CQLWrapper object for use by optimization.
```

Comment:
What is needed "for use by optimization", what is special for this use case?
Why can't you simply use

    new CQLWrapper(new CQL2PgJSON(field), query, limit, offset)

and drop createCQLWrapper completely?